core/commands/keyencode/keyencode.go

```diff
@@ -1,11 +1,14 @@
 package keyencode
```

Comment:
Any ideas why code coverage is yelling at me here? Is this just because I removed some lines, or is there something sneaky that I've missed? The report looks pretty weird.